### PR TITLE
Cross-Origin Resource Sharing PoC

### PR DESCRIPTION
**Pull request by kousha1999**
This PR covers issue #691.

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- PoC for Wildcard Access-Control-Allow-Origin

### CODE REVIEW

**Comment by kousha1999** [comment]
@jeremychoi Hi sir. Can you review my code? @ThunderSon mentioned you before but cause of my mistake in last PR, I had to delete that PR.

File: document/4-Web_Application_Security_Testing/11-Client-side_Testing/07-Testing_Cross_Origin_Resource_Sharing.md [hunk 1]
```diff
@@ -74,6 +74,30 @@ Content-Type: application/xml
 [Response Body]
 ```
 
+If response contains sensitive data, Attacker can steal it by `XMLHttpRequest`. A simple PoC to steal response can be like this:
```
Comment (jeremychoi) [inline document/4-Web_Application_Security_Testing/11-Client-side_Testing/07-Testing_Cross_Origin_Resource_Sharing.md:77]: A few trivial changes for grammar may be needed, a suggestion would be:
"If **a** response contains sensitive data, **an** attacker can steal it by `XMLHttpRequest`. A simple PoC to steal response can be like this":

The code below looks good to me.

**Comment by jeremychoi** [comment]
@ThunderSon Done. Now I'll leave it to you.

**Review by ThunderSon — APPROVED** [review]
Thank you!

**Pull request merged**